# The name of the topic
string topic

# The data
sensor_msgs/CompressedImage data